packages/data-transformation/fhir-to-csv/src/parseFhir/parseFhir.py

```
@@ -0,0 +1,341 @@
+
+
+    if inputFormat == 'ndjson':
+        with open(inputPath, encoding='utf-8-sig') as inputFile:
+            for jsntxt in inputFile:
+                result_count = 0
+                try:
+                    jsndict = json.loads(jsntxt)
+                    result_count = parse_one_resource(anchor, paths, jsndict, leng, csvwriter,data,inputPath,outputFormat)
+                    if missingPath:
+                        compare_and_write_new_paths(jsndict,inputFile,anchor,config,missingPath)
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)